src/track/serato/tags.cpp

```diff
@@ -119,8 +119,11 @@ double SeratoTags::findTimingOffsetMillis(const QString& filePath) {
         // TODO: Find missing timing offsets
         switch (timingShiftCase) {
 #if defined(__MAD__)
+        case EXIT_CODE_CASE_A:
+        case EXIT_CODE_CASE_B:
+        case EXIT_CODE_CASE_C:
         case EXIT_CODE_CASE_D:
```

Comment:
All of these cases are the same? Is it the same offset for *all* MP3 files, even those besides these cases?

Reply:
Possible, I didn't encounter MP3s that didn't fall into one of these categories yet. As I said, I visually matches the position as best as I could, but that's still horribly imprecise. I wish we had a programmatic way to check this.

Reply:
You could hack the deck time display so it is more precise than 10 ms (I don't think that would be useful to add an option for normal Mixxx use). @ehendrikd how did you measure the differences for Rekordbox?

Reply:
I'll check tomorrow but this only helps if Serato's position display is more precise, right?

Reply:
Ah, nevermind. I can just use the raw position values from the metadata tags which are in milliseconds.

Reply:
From memory a very similar process. Started with the quoted 26ms offset, however in practice this was clearly incorrect in some cases, and clearly not multiples of 26. Basically zoomed in waveforms and compared them visually, and tried to confirm by ear as best I could.

Reply:
@ehendrikd, with the modifications you made to mp3guessenc, does every MP3 fall into these 4 exit codes? Or are there MP3s that don't fall into any of these categories?

Reply:
Case D is a catchall if no other case is found:
https://github.com/mixxxdj/mixxx/blob/998cd286d46cfc52b9d6bcd210cfb8327f84a8bc/lib/mp3guessenc-0.27.4/tags.c#L529